erpnext/patches/v8_1/gst_fixes.py

```diff
@@ -39,8 +39,8 @@ def add_custom_fields():
 
 	custom_fields = {
 		'Address': [
-			dict(fieldname='gst_state_number', label='GST State Number',
-				fieldtype='Int', insert_after='gst_state'),
+			dict(fieldname='auto_repeat', label='Auto Repeat',
```

Comment:
Is it done by mistake?

Reply:
Definitely a mistake. I have no idea why I did this. Reverting